### ISSUE TITLE
manpages have escaped second - in --, i.e. like -\-version

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#4522]
Initially spotted on smaug with 0.12.2, upgraded to 0.12.6 - the same. E.g. `man datalad` shows  e.g.
```
       -l LEVEL, -\-log-level LEVEL
              set  logging  verbosity  level. Choose among critical, error, warning, info, debug. Also you can
              specify an integer <10 to provide even more debugging information

       -\-pbs-runner {condor}
```

**Comment by kyleam** [comment]
Should be fixed by #4332.

> yarikoptic added this to the 0.12.7 

While we're of course free to cut 0.12.7 if need be, I don't see much value at this point in tagging stuff with that, especially cosmetic issues like this that have been in several releases.

**Comment by yarikoptic** [comment]
> [duplicate of #4032]
> 
> Should be fixed by #4332.

yeah... found that now too.  It is unfortunate that at that time it was not submitted against `maint`.  I will cherry pick that commit into maint and push directly just in case 0.12.7 sees the light of the release.

> > yarikoptic added this to the 0.12.7
> 
> While we're of course free to cut 0.12.7 if need be, I don't see much value at this point in tagging stuff with that, especially cosmetic issues like this that have been in several releases.

I label PRs for a release largely to later be able to tell in which release any particular PR ended up. For that reason I quite often even label post-release, regardless the level of severity.

**Comment by yarikoptic** [comment]
addressed by cherry picking into b80d930deb2a82e2d9175d8c0cfe45812d2ae5ec

**Comment by kyleam** [comment]
> I label PRs for a release largely to later be able to tell in which release any particular PR ended up. For that reason I quite often even label post-release, regardless the level of severity.

All right.  Noted.
